nntrainer/tensor/manager.h

```diff
@@ -408,8 +419,9 @@ class Manager {
   std::vector<std::vector<std::unique_ptr<Weight>>>
     weights_v2; /**< weights for the layers */
   std::vector<std::vector<std::unique_ptr<Var_Grad>>>
-    inouts_v2; /**<
-              inputs/outputs for the layers */
+    inputs_v2; /**< inputs for the layers */
+  std::vector<std::vector<std::unique_ptr<Var_Grad>>>
+    outputs_v2; /**< outputs for the layers */
```

Comment:
are we going to keep this _v2 later?

Reply:
the names can be updated, that's upto us.
I kept v2 as weights already exist, so created weights_v2.